collie/model/hybrid_matrix_factorization.py

```diff
@@ -162,19 +183,30 @@ def __init__(self,
                  load_model_path: Optional[str] = None,
                  map_location: Optional[str] = None):
         item_metadata_num_cols = None
+        user_metadata_num_cols = None
         optimizer_config_list = None
 
         if load_model_path is None:
-            if item_metadata is None:
-                raise ValueError('Must provide item metadata for ``HybridPretrainedModel``.')
-            elif isinstance(item_metadata, pd.DataFrame):
-                item_metadata = torch.from_numpy(item_metadata.to_numpy())
-            elif isinstance(item_metadata, np.ndarray):
-                item_metadata = torch.from_numpy(item_metadata)
-
-            item_metadata = item_metadata.float()
+            if item_metadata is None and user_metadata is None:
+                raise ValueError(
+                    'Must provide item metadata or user metadata for ``HybridModel``.'
```

Comment:
[Optional] Slight language tweak here: 

```suggestion
                    'Must provide item metadata and/or user metadata for ``HybridModel``.'
```

Reply:
👍